lib/libutils/isoc/bget.c

```diff
@@ -1000,6 +1004,8 @@ void brel(buf, poolset, wipe)
 	poolset->freelist.ql.blink = b;
 	b->ql.blink->ql.flink = b;
 	b->bh.bsize = -b->bh.bsize;
+        wipe_start = (char *)b + sizeof(struct bfhead);
```

Comment:
Prefer add an empty line above.
Maybe add a dedicated inline comment for the added instructions? (`/* Set memory area to be wiped */`)?

Reply:
Fine, I've added an empty line above.